src/track/external-captions-handler.js

```diff
@@ -231,6 +233,38 @@ class ExternalCaptionsHandler extends FakeEventTarget {
     }
   }
 
+  /**
+   * Add cuechange listener to active textTrack.
+   * @param {TextTrack} textTrackEl - selected text track
+   * @returns {void}
+   * @private
+   */
+  addCueChangeListener(textTrackEl: TextTrack): void {
+    this._eventManager.listen(textTrackEl, 'cuechange', (e: FakeEvent) => this._onCueChange(e));
+  }
+
+  /**
+   * Remove cuechange listeners from textTracks
+   * @returns {void}
+   * @private
+   */
+  _removeCueChangeListeners(): void {
+    for (let i = 0; i < this._player.getVideoElement().textTracks.length; i++) {
```

Comment:
need to verify `this._player.getVideoElement()` and `textTracks` existence
like we do in `_addCuesToNativeTextTrack`

Reply:
done